lib-ce/emqx_dashboard/src/emqx_dashboard_admin.erl

```diff
@@ -128,6 +128,10 @@ change_password(Username, OldPasswd, NewPasswd) when is_binary(Username) ->
     end.
 
 change_password(Username, Password) when is_binary(Username), is_binary(Password) ->
+    case binenv(default_user_username) of
+        Username -> application:set_env(emqx_dashboard, default_user_passwd, Password);
```

Comment:
we'll need to call (rpc) other nodes all to set the same password.

change_password sets password in db which is replicated to other nodes
if we change app-env only locally, it's an inconsistent behaviour